x-pack/plugin/eql/src/main/antlr/EqlBase.g4

```diff
@@ -191,14 +191,12 @@ PIPE: '|';
 
 
 ESCAPED_IDENTIFIER
-    : '`' (~'`')* '`'
+    : '\'' (~'`')* '\''
```

Comment:
We haven't agreed on this.

https://github.com/elastic/elasticsearch/pull/62539#pullrequestreview-491521943